pom.xml

```diff
@@ -3,14 +3,14 @@
 
   <groupId>com.avides.springboot.springtainer</groupId>
   <artifactId>springtainer-common</artifactId>
-  <version>1.0.0</version>
+  <version>1.1.0</version>
```

Comment:
Hast du das lokal mal mit den Containern durchgespielt? Wenn nicht, würde ich das noch vorher mal testen oder sonst ggf. einen RC machen. Oder?

Reply:
Version existiert bereits (wahrscheinlich durch die release-Tests). Könnten wir dadurch Probleme beim eigentlichen release bekommen? 
Zudem ist dann noch die Frage, ob in dem Release schon alle Features / Updates drin waren oder sich nachträglich noch verändert haben

Reply:
Habe das getestet und es scheint zu laufen. Die Version habe ich sicherheitshalber nochmal angehoben.